auth/src/App.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import "./App.css";
+import { Route, NavLink } from "react-router-dom";
+import Login from "./Login";
+import Users from "./Users";
+import axios from "axios";
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      url: "http://localhost:5111"
+    };
+  }
+
+  login = user => {
+    console.log("hit login");
+    console.log("user", user);
+    // const axiosObj = { headers: user, body: {} };
+
+    axios.interceptors.request.use(
+      options => {
+        options.headers.username = user.username;
+        options.headers.password = user.password;
+        return options;
+      },
+      err => {
+        return Promise.reject(err);
+      }
+    );
+
+    // axios({
+    //   method: "post",
+    //   url: `${this.state.url}/api/login`,
+    //   headers: user
+    // })
+
+    axios
+      .get(`${this.state.url}/api/login`)
+      .then(res => {
+        console.log(res.data);
```

Comment:
so here you would set the token to your localStorage or state so that you can use it to authenticate on future api calls to protected routes. also, you may want to redirect the user to the users component upon successful login. you may need to use withRouter on your app component to utilize the props.history.push method